.stylelintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   extends: [
-    "stylelint-config-recommended-scss",
-    "@mate-academy/stylelint-config"
+    // "stylelint-config-recommended-scss",
+    // "@mate-academy/stylelint-config"
```

Comment:
Why did you decide to disable these rules?

Reply:
i have a problem with lint:

src/styles/utils/variables.scss
 1:1  ✖  Unknown rule annotation-no-unknown                               annotation-no-unknown                  
 1:1  ✖  Unknown rule function-no-unknown. Did you mean unit-no-unknown?  function-no-unknown                    
 1:1  ✖  Unknown rule custom-property-no-missing-var-function             custom-property-no-missing-var-function
 1:1  ✖  Unknown rule keyframe-block-no-duplicate-selectors               keyframe-block-no-duplicate-selectors  
 1:1  ✖  Unknown rule selector-anb-no-unmatchable